### ISSUE TITLE
replace '{{value}}' with SunConstants.VALUE_NAMED_PLACEHOLDER

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#471]
Noted in https://github.com/phetsims/scenery-phet/issues/446.

Please replace string literal '{{value}}' with NumberDisplay.NAMED_PLACEHOLDER in AccessibleSlider and AccessibleNumberSpinner.

**Comment by pixelzoom** [comment]
This string constant was moved to SunConstants.VALUE_NAMED_PLACEHOLDER in https://github.com/phetsims/sun/issues/474.

Searching for "{{value}}", here are the places where this it should be replaced with SunConstants.VALUE_NAMED_PLACEHOLDER:

- [ ] SUN/AccessibleSlider
- [ ] SUN/AccessibleNumberSpinner
- [ ] OHMS_LAW/SliderUnit
- [ ] RESISTANCE_IN_A_WIRE/SliderUnit

**Comment by jessegreenberg** [comment]
All four of the above instances of "{{value}}" have been moved to SUN/AccessibleValueHandler as part of https://github.com/phetsims/scenery/issues/951 so I will replace that one.
